addons/l10n_it_edi/models/account_move.py

```diff
@@ -0,0 +1,458 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import logging
+import re
+
+from odoo import api, fields, models, _
+from odoo.tools import float_compare, float_repr
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_state = fields.Selection([
+        ('being_sent', 'Being Sent To SdI'),
+        ('needs_signing', 'Needs signing'),
+        ('processing', 'SdI Processing'),
+        ('rejected', 'SdI Rejected'),
+        ('forwarded', 'SdI Accepted, Forwarded to Partner'),
+        ('forward_failed', 'SdI Accepted, Forward to Partner Failed'),
+        ('forward_attempt_failed', 'SdI Accepted, Forward Attempt Failed'),
+        ('accepted_by_pa_partner', 'SdI Accepted, Accepted by the PA Partner'),
+        ('rejected_by_pa_partner', 'SdI Accepted, Rejected by the PA Partner'),
+        ('accepted_by_pa_partner_after_expiry', 'SdI Accepted, PA Partner Expired Terms'),
+    ], copy=False, tracking=True, readonly=True)
+    l10n_it_edi_transaction = fields.Char(copy=False, string="FatturaPA Transaction")
+    l10n_it_edi_attachment_file = fields.Binary(copy=False, attachment=True)
+    l10n_it_edi_attachment_id = fields.Many2one(
```

Comment:
Yes, it will be necessary. 
The Binary vs stored Many2One attachment requirement is kinda new, I asked LAS to review to check it out 'cause he already implemented it, if it's OK I can make the upgrade.